### PR TITLE
Adding Glyphicons to the UI

### PR DESCRIPTION
**Pull request by Guntrisoft**
Adding iconography to improve the UI

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -27,8 +27,8 @@
   <tbody>
   <% @schedules.each do |schedule| %>
     <tr>
-      <td><%= "#{schedule.title}" %></td>
-      <td><%= link_to edit_user_schedule_path(@guider, schedule), title: "Edit Schedule #{schedule.title}", :class => 'btn btn-info' do %><span class="glyphicon glyphicon-pencil"></span><span class="sr-only">Edit Schedule</span><% end %></td>
+      <td><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> <%= "#{schedule.title}" %></td>
+      <td><%= link_to edit_user_schedule_path(@guider, schedule), title: "Edit Schedule #{schedule.title}", :class => 'btn btn-info' do %><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span><span class="sr-only">Edit Schedule</span><% end %></td>
```
Comment (benlovell) [inline app/views/users/edit.html.erb]: Use 1.9 style hash key / value for `class`.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -25,8 +25,8 @@
       <% @guiders.each do |guider| %>
       <tr data-multi-checkbox-group>
         <td><label for="guider_<%= guider.id %>"><input data-multi-checkbox="item" type="checkbox" name="guider_<%= guider.id %>" id="guider_<%= guider.id %>"><span class="sr-only">Select <%= guider.name %></span></label></td>
-        <td class="name"><%= guider.name %></td>
-        <td><%= link_to edit_user_path(guider), title: "Edit #{guider.name}", :class => 'btn btn-info' do %><span class="glyphicon glyphicon-pencil"></span><span class="sr-only">Edit Guider</span><% end %></td>
+        <td class="name"><span class="glyphicon glyphicon-user" aria-hidden="true"></span> <%= guider.name %></td>
+        <td><%= link_to edit_user_path(guider), title: "Edit #{guider.name}", :class => 'btn btn-info' do %><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span><span class="sr-only">Edit Guider</span><% end %></td>
```
Comment (benlovell) [inline app/views/users/index.html.erb]: Same here.

File: app/views/users/edit.html.erb (hunk 1 above)
Comment (Guntrisoft) [inline app/views/users/edit.html.erb]: damn it, always catches me out

File: app/views/groups/index.html.erb [hunk 3]
```diff
@@ -27,12 +27,15 @@
         <tbody>
           <% @groups.each do |group| %>
             <tr class="t-group">
-              <td><%= group.name %></td>
-              <td><%= link_to 'Delete',
-                    group, method: :delete,
-                    class: 'btn btn-danger t-delete',
-                    data: { confirm: "Are you sure you want to delete the group #{group.name}?" }
+              <td><span class="glyphicon glyphicon-tag" aria-hidden="true"></span> <%= group.name %></td>
+              <td><%= link_to group,
```
Comment (AndrewVos) [inline app/views/groups/index.html.erb]: Can this be added to the next line or is it a whitespace issue?

**Comment by Guntrisoft** [comment]
@benlovell @AndrewVos changes made - happy to merge?

**Pull request merged**